src/ex1/script.js

```diff
@@ -0,0 +1,91 @@
+let addTaskBtn = document.querySelector(".submitBtn")
+let inputText = document.querySelector(".taskText")
+
+//Handle Image
+function addImg() {
```

Comment:
Theres a funny bug-
when all the items are selected, if you delete one of them - it will append the empty state image over and over
![image](https://user-images.githubusercontent.com/36815567/169701951-90f6c802-b482-4eeb-a6d9-f4d4c6f38c63.png)